### PR TITLE
Tutorial Updates

### PR DESCRIPTION
**Pull request by abachma2**
The tutorial has been updated based on some comments received already. Data exploration and analysis has been replaced with directions to look at Cymetric and its documentation.

### CODE REVIEW

**Comment by abachma2** [comment]
I don't want to merge this until I have a chance to walk through it with some people. But feel free to build the pages yourself and take a look

File: source/user/tutorial/add_proto.rst [hunk 1]
```diff
@@ -472,15 +469,16 @@ This facility takes two inputs, ``name`` and ``outcommd``. Using the Source Arch
     </config>
   </facility>
 
-Once complete, append this facility under the recipe section of your input file.
+Once complete, append this facility under the commodity section of your input file.
```
Comment (robfairh) [inline source/user/tutorial/add_proto.rst:472]: I think I would leave it as 'recipe section.'
Also, the page refers to the 'recipe section' more than once.
For example, line 167.
So if you change it here, you should change it there as well.
I wouldn't change it.

```suggestion
Once complete, append this facility under the recipe section of your input file.
```

File: source/user/tutorial/add_second_reactor.rst [hunk 2]
```diff
@@ -116,85 +117,5 @@ of the region block should now look like,
 Note: the blank space between ``</institution>`` and ``</region>`` is
 for additional institutions in the future.
 
-Save your input file as input_file2.xml and run the cyclus simulation.
+Save your input file as input_file2.xml and run the |Cyclus| simulation.
 
-Activity: Analysis
```
Comment (robfairh) [inline source/user/tutorial/add_second_reactor.rst:121]: Why was this deleted?

**Comment by robfairh** [comment]
> The tutorial has been updated based on some comments received already.

Could you add some more information about this?

File: source/user/tutorial/add_second_reactor.rst (hunk 2 above)
Comment (abachma2) [reply source/user/tutorial/add_second_reactor.rst:121]: This was an extension from data_explorer.rst, both using an analysis.py file. @katyhuff and I both agreed that any analysis should be changed to using cymetric, since that is a more supported tool. For the time being, I have removed the analysis from the tutorial and point the user to the guide on cymetric on the website and the examples in the cymetric github (github.com/cyclus/cymetric). I can always add a data analysis section for the tutorial back in later once the core is redone for xml input files.

**Comment by abachma2** [comment]
The previous comments come from Paul Wilson on the PR from this branch to the cyclus/cyclus.github.com repo (PR#293) https://github.com/cyclus/cyclus.github.com/pull/293. They mostly involve the distinction of the different aspects of the cyclus architecture (e.g. archetypes vs prototypes)

File: source/user/tutorial/add_second_reactor.rst (hunk 2 above)
Comment (robfairh) [reply source/user/tutorial/add_second_reactor.rst:121]: That makes sense!

File: source/user/tutorial/install_launch_cyclus.rst [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
 
-For this tutorial, we only require the Cyclus, Cycamore, and Cymetric. You can
-install Cyclus using the following steps listed on the `Install Cyclus <http://fuelcycle.org/user/install.html>`_ page.
-You can install Cycamore from the `Cycamore github repository <https://github.com/cyclus/cycamore>`_. 
-You can install Cymetric from the `Cymetric github repository <https://github.com/cyclus/cymetric>`_. 
+For this tutorial, the |Cyclus|, and Cycamore programs are utilized. You can
+install |Cyclus| using the following steps listed on the `Install Cyclus <http://fuelcycle.org/user/install.html>`_ page.
+You can install Cycamore from the `Cycamore github repository <https://github.com/cyclus/cycamore>`_. It is not required to 
+install these programs becuase there is a cloud service available to running |Cyclus| input files, as discussed 
+`here <https://fuelcycle.org/user/launch_cyclus.html>`_.
```
[the remark this answers is not in the document]
Comment (abachma2) [reply source/user/tutorial/install_launch_cyclus.rst:8]: Which link are you referring to?

[the remark this answers is not in the document]
Comment (abachma2) [reply source/user/tutorial/install_launch_cyclus.rst:8]: The <https://fuelcycle.org/user/launch_cyclus.html> link is a new page that is not part of the website right now, but will be when this all gets pulled into cyclus/cyclus.githu.com. You can build the page using `make gh-preview` in the directory, or I can show it to you when I run through the tutorial.

[the remark this answers is not in the document]
Comment (robfairh) [reply source/user/tutorial/install_launch_cyclus.rst:8]: I am referring to https://fuelcycle.org/user/launch_cyclus.html.
If you see the changes in the 'Files changed' tab in the PR, you should be able to see which line my comments refer to.

[the remark this answers is not in the document]
Comment (robfairh) [reply source/user/tutorial/install_launch_cyclus.rst:8]: Okay, that makes sense! Thanks for the clarification.

File: source/user/tutorial/install_launch_cyclus.rst [hunk 4]
```diff
@@ -1,8 +1,10 @@
 Installing and Launching Cyclus
 ===============================
 
-For this tutorial, we only require the Cyclus, Cycamore, and Cymetric. You can
-install Cyclus using the following steps listed on the `Install Cyclus <http://fuelcycle.org/user/install.html>`_ page.
-You can install Cycamore from the `Cycamore github repository <https://github.com/cyclus/cycamore>`_. 
-You can install Cymetric from the `Cymetric github repository <https://github.com/cyclus/cymetric>`_. 
+For this tutorial, the |Cyclus|, and Cycamore programs are utilized. You can
+install |Cyclus| using the following steps listed on the `Install Cyclus <http://fuelcycle.org/user/install.html>`_ page.
+You can install Cycamore from the `Cycamore github repository <https://github.com/cyclus/cycamore>`_. It is not required to
```
[the remark this answers is not in the document]
Comment (abachma2) [reply source/user/tutorial/install_launch_cyclus.rst:6]: It is not only for the tutorial. It is available here: http://fuelcycle.org/#try-it

[the remark this answers is not in the document]
Comment (robfairh) [reply source/user/tutorial/install_launch_cyclus.rst:6]: Cool stuff

**Comment by robfairh** [comment]
looks good, merging
